assets/styl/_theme/_skin.styl

```diff
@@ -135,6 +152,7 @@
 
     /* Icon skins */
     &-gallery
+    &-podcast
     &-video
         .media > p:first-child
```

Comment:
Since we reworked `media`, we should use `.media-wrap` instead of `.media > p:first-child, .media > div:first-child`